src/System.Windows.Extensions/ref/System.ComponentModel.TypeConverter.cs

```diff
@@ -0,0 +1,67 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
This should not be a new file. This should be in the existing `System.Windows.Extensions.cs`. The convention is for ref folder to have just one `.cs` file with everything.